src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -5167,6 +5182,32 @@ namespace EnergyPlus::OutputReportTabular {
         print(state.files.audit, variable_fmt, "numCompSizeTableEntry", state.dataOutRptPredefined->numCompSizeTableEntry);
     }
 
+    bool produceDualUnitsFlags(const int& iUnit_Sys, const iUnitsStyle& unitsStyle_Tab, const iUnitsStyle& unitsStyle_Sql, 
```

Comment:
Please add some comments on what this function is doing and how it is used.

Reply:
> 
> 
> Please add some comments on what this function is doing and how it is used.

Adding

Reply:
This one is done.